sqlmesh/core/model/seed.py

```diff
@@ -24,6 +25,16 @@ class Seed(PydanticModel):
     def columns_to_types(self) -> t.Dict[str, exp.DataType]:
         return columns_to_types_from_df(self._get_df())
 
+    @property
+    def column_hashes(self) -> t.Dict[str, str]:
+        df = self._get_df()
+        return {
+            column_name: hashlib.md5(  # type: ignore
```

Comment:
let's use crc, it's faster, non cryptographic, and avoids collisions on close text

Reply:
md5 has long been considered to be non-cryptographic lol. TBH I don't like our choice of crc even for snapshot fingerprinting. We unnecessarily expose ourselves to the risk of collision. I'll make a change and will wait for one